eng/validation/update-channel.ps1

```diff
@@ -0,0 +1,47 @@
+Param(
+  [string] $maestroEndpoint,
+  [string] $barToken,
+  [int] $channelId = 2             # channel 2 maps to  ".NET Tools - Latest" in Prod
+)
+
+. $PSScriptRoot\..\common\tools.ps1
```

Comment:
What are we getting from tools.ps1?

Reply:
It loads `$GlobalJson` with the contents to `global.json`